Index: erpProjectVVS/src/main/java/realERPproject/erpProjectVVS/order/service/OrderService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package realERPproject.erpProjectVVS.order.service;\r\n\r\nimport lombok.RequiredArgsConstructor;\r\nimport org.springframework.http.ResponseEntity;\r\nimport org.springframework.stereotype.Service;\r\nimport realERPproject.erpProjectVVS.common.response.ApiResponse;\r\nimport realERPproject.erpProjectVVS.order.dto.OrderRequest;\r\nimport realERPproject.erpProjectVVS.order.dto.OrderResponse;\r\nimport realERPproject.erpProjectVVS.order.repository.OrderRepository;\r\n\r\n@Service\r\n@RequiredArgsConstructor\r\npublic class OrderService {\r\n\r\n    private final OrderRepository orderRepository;\r\n\r\n    public ResponseEntity<ApiResponse<OrderResponse>> createOrder(OrderRequest orderRequest){\r\n\r\n        orderRepository.save(orderRequest);\r\n\r\n        return \r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/erpProjectVVS/src/main/java/realERPproject/erpProjectVVS/order/service/OrderService.java b/erpProjectVVS/src/main/java/realERPproject/erpProjectVVS/order/service/OrderService.java
--- a/erpProjectVVS/src/main/java/realERPproject/erpProjectVVS/order/service/OrderService.java	(revision 5a67576fec533cfe5878c65e022d704b781434af)
+++ b/erpProjectVVS/src/main/java/realERPproject/erpProjectVVS/order/service/OrderService.java	(date 1757599137542)
@@ -18,6 +18,6 @@
 
         orderRepository.save(orderRequest);
 
-        return 
+        return
     }
 }
